recipes/augustus/3.2.3_boost1.57/patches/filterBam.src.Makefile.patch

```diff
@@ -12,8 +12,9 @@
 +INCLUDES =  -I${PREFIX}/include/bamtools/ -Iheaders -I./bamtools
 +#LIBS = -lbamtools -lz
 +LIBS = ${PREFIX}/lib/libbamtools.a -lz
- CFLAGS = -std=c++0x
+-CFLAGS = -std=c++0x
++CFLAGS = -std=c++11
 +CXXFLAGS += -Wall -O2 # -g -p -g -ggdb 
```

Comment:
We could either remove
```diff
+CXXFLAGS += -Wall -O2 # -g -p -g -ggdb
```
from the patch or use
```diff
-CFLAGS = -std=c++0x
+CFLAGS += -std=c++11 -Wall -O2
```
instead. Currently, setting `CXXFLAGS` is a no-op since the `Makefile` doesn't adhere to convention and uses `CFLAGS` instead of `CXXFLAGS`.

Reply:
What’s the `-O2` doing in `CXXFLAGS`? If the `Makefile` doesn’t use them, are they actually doing anything? Thanks for the catch.

Reply:
> If the `Makefile` doesn’t use them, are they actually doing anything?

Nope, should be a no-op that we can remove, I think